sdk/keyvault/keyvault-secrets/review/keyvault-secrets.api.md

```diff
@@ -137,12 +137,14 @@ export interface SecretPollerOptions extends coreHttp.OperationOptions {
 
 // @public
 export interface SecretProperties {
+    readonly certificateKeyId?: string;
     contentType?: string;
     readonly createdOn?: Date;
     enabled?: boolean;
     readonly expiresOn?: Date;
     id?: string;
-    readonly keyId?: URL;
+    // @deprecated
+    readonly keyId?: any;
```

Comment:
small preference for making this `unknown` instead

Reply:
Yeah, I started with `unknown` but then I thought what if someone accidentally did something like 

```
let myUrl: URL = keyId
```

If we changed it to unknown, that will break at compile time, but on the other hand it would have never worked at runtime anyway so maybe that's just FUD (Fear, Uncertainty, Doubt) on my part and actually better if it alerts at compile time?

Reply:
A lot of folks infer the type when using direct assignment, so I'm not too worried about this, especially if the property was never set before. I don't have strong feelings if you think it's safer to use `any`, though.

Reply:
I like unknown too, I think I just needed some assurance 😄 

I switched back to unknown